torch/testing/_internal/common_methods_invocations.py

```diff
@@ -4215,6 +4215,7 @@ def sample_inputs_interpolate(mode, self, device, dtype, requires_grad, **kwargs
         align_corners_options = (True, False, None)
     ranks_for_mode = {
         'nearest': [1, 2, 3],
+        'nearest_exact': [1, 2, 3],
```

Comment:
seems like this should be 
```suggestion
        'nearest-exact': [1, 2, 3],
```